src/libs/dal/Configuration/ServerItemConfiguration.cs

```diff
@@ -15,6 +15,7 @@ public override void Configure(EntityTypeBuilder<ServerItem> builder)
         builder.Property(m => m.TenantId);
         builder.Property(m => m.OrganizationId);
         builder.Property(m => m.OperatingSystemItemId);
+        builder.Property(m => m.HistoryKey);
```

Comment:
This is essentially a foreign key, but it's not enforced otherwise we'd have a chicken/egg issue.